windows/win_iis_webapppool.ps1

```diff
@@ -90,10 +90,14 @@ try {
       Stop-WebAppPool -Name $name -ErrorAction Stop
       $result.changed = $TRUE
     }
-    if ((($state -eq 'started') -and ($pool.State -eq 'Stopped')) -or ($state -eq 'restarted')) {
+    if ((($state -eq 'started') -and ($pool.State -eq 'Stopped'))) {
       Start-WebAppPool -Name $name -ErrorAction Stop
       $result.changed = $TRUE
     }
+    if ($state -eq 'restarted') {
+      Restart-WebAppPool -Name $name
```

Comment:
The logic behind this code change is to provide the consumers of this module the ability to recycle an apppool that may be in a bad state or after a config change.
What's the difference between restarted and started? The logic added with this commit is to recycle the apppool (restart) if it is already running. If "restarted" and "started" do the same thing, what is the need for 2 states? Would it make a difference if the state "recycle" was used in place of restarted on line 97?